content/chainguard/chainguard-images/unique-tags/index.md

```diff
@@ -0,0 +1,70 @@
+    parent: "chainguard-images"
+weight: 735
+toc: true
+---
+
+Some organizations use image tags as an indication that there is a new container image available in a registry. Oftentimes, these organizations' internal automation and policies expect each new build to have its own distinct tag. 
+
+To help with cases like this, Chainguard offers Unique Tags for private registries. Unique Tags are ideal for organizations that require a strict tag per release or update of their images. They benefit teams looking for precise tracking and management of container images. 
+
+This guide provides an overview of what these Unique Tags are, the kinds of problems they aim to solve, and how you can access them in the Chainguard Console.
+
+
+## Chainguard's Unique Tags
+
+Unique Tags are only available for private registries, as the public Chainguard Registry only has the `:latest` or, in some cases, the `:latest-dev` tags available. Unique Tags feature an opt-in feature, which allows customers the flexibility to enable it based on their specific operational and security requirements.
+
+Chainguard's Unique Tags end in a timestamp, such as `20240229`, which indicates the date when the Image was built. Because Chainguard Images are rebuilt daily, this ensures that the specific tag will always represent that specific Image build and not another.
+
+One benefit of using this timestamp scheme with Unique Tags is that it can help users to quickly identify when a given version of an Image was built. It also helps to make them human-readable, as opposed to the long, unpronounceable strings that make up Image digests.
+
+Unique Tags also allow for individual image repositories within a registry to be included or excluded as needed. For example, if you have an application that requires a specific tagging scheme to be compatible with an existing Helm chart, you can enable Unique Tags for your registry, but exclude that specific repository so that its Images only receive the standard tags.
+
+This granular level of control ensures that organizations can implement unique tagging in a way that best suits their organization's specific needs. It offers a tailored approach to image management, allowing for precise and efficient tracking of image versions and builds across different environments.
+
+Additionally, be aware that the Unique Tags feature is integrated with Chainguard's [Tag History](/chainguard/chainguard-images/using-the-tag-history-api/) and [Diff APIs](/chainguard/chainguard-images/using-the-image-diff-api/), as well as the [`chainctl diff` command](/chainguard/chainguard-images/comparing-images/) and [event notifications](/chainguard/administration/cloudevents/events-reference/). These integrations allow you to track changes, compare different image versions, and understand the differences between various builds. It empowers organizations to maintain a detailed and accessible record of their container images, facilitating easier analysis and decision-making.
```

Comment:
Maybe drop "be aware" and have it read "Additionally, the Unique Tags feature...". Only because I feel like be aware is a warning I might get something wrong or need to implement some kind of check or functionality to use this?